src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+
+export enum FilterTypes {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
+
+type Props = {
+  todos: Todo[]
+  onFilterType: (e: any) => void
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  todos,
+  onFilterType,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${todos.length} items left`}
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
+      <nav className="filter">
+        <button
+          type="button"
+          // href="#/"
```

Comment:
use it (should be a link with displayed selected filter in the URL), also fix styles - doesn't match the working example:
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/86923089/f6ab46fd-0e95-47af-bca6-03784552da92)
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/86923089/d214ee23-6048-4c32-94ba-7cf487cf1cd7)